docs/source/usage/particles.rst

```diff
@@ -23,6 +23,8 @@ DeriveSpecies
 Manipulate
 ^^^^^^^^^^
 
+Some of the particle manipulators further take the functors and filter (below) as arguments to manipulate attributes of particle species.
```

Comment:
Add on next line:
A particle filter is used to only manipulated selected particles of a species with a functor.